.github/workflows/ci.yml

```
@@ -27,11 +27,15 @@ jobs:
           node-version: 14.x
           registry-url: https://npm.pkg.github.com/
 
+      - uses: actions/checkout@v2
+
+      - name: npmrcgen
```

```suggestion
      - name: Create npmrc file
```
Also, add this whole step in `release.yml` after the `checkout` step.